File: progetto/testo.cpp
#include "testo.h"
#include<QChar>

const ushort testo::alphSize=26;

testo::testo(QString s):text(s){}

//___SETTER___
void testo::setText(QString s){text.clear();text.append(s);}
//___GETTER___
QString testo::getText() const{ return text; }
QChar testo::getLetter(int i) const { return text.at(i); }
int testo::getSize() const{ return text.size(); }

void testo::reset(){ text.clear(); }

File: progetto/java/progettoJavaCipher/src/Testo.java
//import java.lang;

public abstract class Testo{
    private String text;
    public static short alphaSize=26;

    public Testo(String t) {
        text =new String(t);

    }


    public void setText(String s){text=s;}

    //SETTER
    /*public void setText(string s){
        text=new string(s);
    }*/

    //GETTER
    public String getText(){return text;}
    public Character getLetter(int i){return text.charAt(i);}
    public int getSize(){return text.length();}

    //___METODI___
    public abstract void converti();
    public abstract void reset();
}